api/api.go

```diff
@@ -0,0 +1,28 @@
+// Copyright 2016 The Hugo Authors. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package api
+
+import (
+	"github.com/spf13/hugo/commands"
+	"github.com/spf13/viper"
+)
+
+func Build(flags []string) {
+	commands.Execute(flags)
```

Comment:
@bep that would be a good idea, but to do so I would have to export HugoCmd, wouldn't I?